xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -43,7 +43,7 @@ CPVRTimerInfoTag::CPVRTimerInfoTag(bool bRadio /* = false */) :
   m_bFullTextEpgSearch(false)
 {
   m_iClientId           = g_PVRClients->GetFirstConnectedClientID();
-  m_iClientIndex        = -1;
+  m_iClientIndex        = static_cast<unsigned int>(-1);
```

Comment:
In my opinion the best initial value would be 0 here, same as iParentClientIndex.